app/src/main/java/com/github/wnder/ImageFromGallery.java

```diff
@@ -0,0 +1,70 @@
+package com.github.wnder;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class ImageFromGallery extends AppCompatActivity {
```

Comment:
Rename class to keep consistency with the other classes ("ImageFromGalleryActivity")